xds/internal/testutils/e2e/clientresources.go

```diff
@@ -277,6 +278,107 @@ func DefaultServerListener(host string, port uint32, secLevel SecurityLevel) *v3
 	}
 }
 
+// ServerListenerWithRBACHTTPFilters returns an xds Listener resource with HTTP Filters defined in the HCM, and a route
+// configuration that always matches to a route and a VH.
+func ServerListenerWithRBACHTTPFilters(host string, port uint32, rbacCfgs []*rpb.RBAC) *v3listenerpb.Listener {
```

Comment:
Move this to `xds_server_integration_test.go` if we don't expect it to be used by other tests?